INFO:fit_demographic_model.py:Beginning execution of fit_demographic_model.py in directory /u/project/ngarud/jonmah/demo_for_ccgb/Scripts

INFO:fit_demographic_model.py:Progress is being logged to ../Analysis/Bacteroides_uniformis_57318/log.log

INFO:fit_demographic_model.py:Parsed the following arguments:
	outprefix = ../Analysis/Bacteroides_uniformis_57318/
	syn_input_sfs = ../Data/Bacteroides_uniformis_57318_syn.sfs

WARNING:Spectrum_mod:Creating Spectrum with data_folded = True, but mask is not True for all entries which are nonsensical for a folded Spectrum.
Traceback (most recent call last):
  File "fit_demographic_model.py", line 394, in <module>
    DemographicInference().main()
  File "fit_demographic_model.py", line 290, in main
    syn_data = dadi.Spectrum.from_file(syn_input_sfs)
  File "/u/home/j/jonmah/.conda/envs/jonmah_env/lib/python2.7/site-packages/dadi/Spectrum_mod.py", line 256, in from_file
    pop_ids=pop_ids)
  File "/u/home/j/jonmah/.conda/envs/jonmah_env/lib/python2.7/site-packages/dadi/Spectrum_mod.py", line 111, in __new__
    raise ValueError('pop_ids must be of length equal to '
ValueError: pop_ids must be of length equal to dimensionality of Spectrum.
